packages/cli/src/cms/commands/utils.ts

```diff
@@ -0,0 +1,49 @@
+export async function wait(ms: number): Promise<void> {
```

Comment:
Does it make sense to reuse this function [here](https://github.com/Redocly/redocly-cli/blob/2f285063acc46497aab062f97fd9ccc9351e418c/packages/core/src/utils.ts#L268)? Also, I'd use another name, maybe `delay`? What do you think?